chucks-pick3-mobile/src/app/components/generate-draw-time-card/generate-draw-time-card.component.ts

```diff
@@ -207,7 +213,7 @@ export class GenerateDrawTimeCardComponent implements OnInit, OnDestroy {
         });
```

Comment:
Looks like we don't need else block any more.

Reply:
Else block was not found in code.